packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -1383,7 +1383,8 @@ export class EditableGrid extends PureComponent<EditableGridProps, EditableGridS
 
         if (showAsTab) {
             const bulkDisabled = selected.size === 0;
-            const gridDisabled = editorModel.rowCount > maxRows;
+            // const gridDisabled = editorModel.rowCount > maxRows;
+            const gridDisabled = true;
```

Comment:
I assume this hardcoding of `gridDisabled = true` was for testing. Should it be reverted back?
Also, the variable name should probably be updated since it doesn't deal with disable/enable anymore. Maybe `hideGrid` or the inverse of `showGrid`.

Reply:
oops!